src/spyglass/lfp/analysis/v1/lfp_band.py

```diff
@@ -333,36 +333,16 @@ def make(self, key):
             + str(lfp_band_sampling_rate)
             + "Hz"
         )
-        tmp_valid_times = (
-            IntervalList
-            & {
+        interval_key = IntervalList().cautious_insert1(
+            key={
                 "nwb_file_name": key["nwb_file_name"],
                 "interval_list_name": key["interval_list_name"],
-            }
-        ).fetch("valid_times")
-        if len(tmp_valid_times) == 0:
-            lfp_band_valid_times = interval_list_censor(
-                lfp_band_valid_times, new_timestamps
-            )
-            # add an interval list for the LFP valid times
-            IntervalList.insert1(
-                {
-                    "nwb_file_name": key["nwb_file_name"],
-                    "interval_list_name": key["interval_list_name"],
-                    "valid_times": lfp_band_valid_times,
-                    "pipeline": "lfp band",
-                }
-            )
-        else:
-            lfp_band_valid_times = interval_list_censor(
-                lfp_band_valid_times, new_timestamps
-            )
-            # check that the valid times are the same
-            assert np.isclose(
```

Comment:
The cautious insert could still create errors without doing this explicit check.  
- Say that this interval list name has been previously inserted.  
- the band was deleted the interval list stays
- parameters were changed in the band that results in a slightly different interval list
- cautious_insert finds no matches on times, tries to insert with this name
    - We would want an error in the case that it's different